addons/html_builder/static/tests/custom_tab/builder_components/builder_range.test.js

```diff
@@ -37,7 +37,7 @@ test("should commit changes", async () => {
     input.dispatchEvent(new Event("change"));
     await delay();
 
-    expect.verifySteps(["customAction 50", "customAction 50"]);
+    expect.verifySteps(["customAction 50"]);
```

Comment:
Only one as with the callOperation, there is only the one of the onchange and not the one of the onInput.